soccer/Robot.hpp

```diff
@@ -109,6 +113,25 @@ class Robot: public RobotPose
 		return *this == other;
 	}
 
+	std::string to_string() const {
```

Comment:
I think that a good convention would be a good idea. 

Isn't the nonmember to_string() a convention we could use?

Also most cases should be covered by overloading the operator<<